src/thread_manager.ts

```diff
@@ -7,6 +7,20 @@ import type { DefaultGenerics, Event, ExtendableGenerics, OwnUserResponse, Query
 
 const DEFAULT_CONNECTION_RECOVERY_THROTTLE_DURATION = 1000;
 const MAX_QUERY_THREADS_LIMIT = 25;
+export const INITIAL_STATE = {
```

Comment:
```suggestion
const INITIAL_STATE = {
```
I'd keep it private for now (name collision could be also an issue).

Reply:
Yeah, it was my initial instinct as well but I wanted to actually compare against this in the tests to make sure we have no leftovers - I could do the check manually but then I think the test would be too easy to break if we add additional stuff to the state at some point. How do you propose we proceed ?

Reply:
Ah -  then `THREAD_MANAGER_INITIAL_STATE` should do; it's a mouthfull but works well. :)